docs/assets/templates/locationID_template.json

```diff
@@ -4,6 +4,8 @@
 		{
 			"name": "Klufterbach",
 			"id": "KB",
+			"prefix": "Kluf-",
+			"prefixDigitPadding": 4,
```

Comment:
Shouldn't the digit padding here should be 5 because "-" is an additional character?

Reply:
The digit padding is about the numbers that follow the prefix, not the prefix itself. so it's Kluf-####. So if you expect to have under 1000 animals (common), you can use the default of 3 which will top out at Kulf-999

Reply:
![image](https://github.com/user-attachments/assets/faf7e1a5-2c52-4231-8daf-66499c816eb6)
example from a working platform and the related json
![image](https://github.com/user-attachments/assets/b77b0f3a-6f24-438b-a004-4d7feea0b406)